content/en/serverless/insights/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Serverless Insights
+kind: documentation
+---
+
+{{< img src="serverless/serverless_actionable_insights_detail.png" alt="Auto-generated insights for your serverless applications" >}}
+
+Datadog automatically generates suggestions to resolve errors, performance problems, and optimize cost for your serverless applications.
+
+## Setup
+
+Datadog uses AWS CloudWatch metrics, Datadog enhanced AWS Lambda metrics, and Lambda `REPORT` logs to suggest insights to you. To set these up,
+
+ 1. Set up the [Amazon Web Services][1] integration.
+ 2. Set up the [Datadog Forwarder][2] and ensure your Lambda `REPORT` logs are indexed in Datadog.
+ 3. Enable [Enhanced Lambda Metrics][3] for your functions.
+
+**Note**: Datadog generates [High Errors](#high-errors), [High Duration](#high-duration), [Throttled](#throttles), and [High Iterator Age](#high-iterator-age) insights out of the box after setting up the [AWS integration][1]. All other insights, including those generated on individual invocations, require the [Datadog Forwarder][2] and [Enhanced Lambda Metrics][3].
+
+## Generated Insights
+
+### Errors
+
+More than 1% of the function's invocations were errors in the selected time range.
+
+**Resolution:** Examine the function's logs, check for recent code or configuration changes with [Deployment Tracking][4], or look for failures across microservices with [distributed tracing][5].
+
+### High Errors
+
+More than 10% of the function's invocations were errors in the selected time range.
+
+**Resolution:** Examine the function's logs, check for recent code or configuration changes with [Deployment Tracking][4], or look for failures across microservices with [distributed tracing][5].
+
+### High Memory Usage
+
+At least one invocation in the selected time range used over 95% of the allocated memory.
+
+[Distributed tracing][5] can help you pinpoint parts of your application using excessive amounts of memory.
```

Comment:
Technically I don't think this insight identifies that the application is using excessive amounts of memory. It might also be the case that the function is using a perfectly fine amount of memory, but the configured amount of memory is too low. I.e. the problem could be the limit, not the function.